terraform/cyhy_nmap_ec2.tf

```diff
@@ -113,27 +118,401 @@ module "cyhy_nmap_ansible_provisioner_0" {
     "--ssh-common-args='-o StrictHostKeyChecking=no -o ProxyCommand=\"ssh -W %h:%p -o StrictHostKeyChecking=no -q ${var.remote_ssh_user}@${aws_instance.cyhy_bastion.public_ip}\"'"
   ]
   envs = [
-    "host=${aws_instance.cyhy_nmap.0.private_ip}",
+    "host=${aws_instance.cyhy_nmap.*.private_ip[0]}",
     "bastion_host=${aws_instance.cyhy_bastion.public_ip}",
     "host_groups=cyhy_runner"
   ]
   playbook = "../ansible/playbook.yml"
   dry_run = false
 }
 
-# module "cyhy_nmap_ansible_provisioner_1" {
-#   source = "github.com/cloudposse/tf_ansible"
-#   #count = "${local.nmap_instance_count}"
-#
-#   arguments = [
-#     "--user=${var.remote_ssh_user}",
-#     "--ssh-common-args='-o StrictHostKeyChecking=no -o ProxyCommand=\"ssh -W %h:%p -o StrictHostKeyChecking=no -q ${var.remote_ssh_user}@${aws_instance.cyhy_bastion.public_ip}\"'"
-#   ]
-#   envs = [
-#     "host=${aws_instance.cyhy_nmap.1.private_ip}",
-#     "bastion_host=${aws_instance.cyhy_bastion.public_ip}",
-#     "host_groups=cyhy_runner"
-#   ]
-#   playbook = "../ansible/playbook.yml"
-#   dry_run = false
-# }
+module "cyhy_nmap_ansible_provisioner_1" {
```

Comment:
Maybe it's time to think about a pre-processor for the provisioners.  Getting too 💦WET💦.  Need to 🍸DRY🌵 out.